Formula/microplane.rb

```diff
@@ -0,0 +1,39 @@
+class Microplane < Formula
+  desc "CLI tool to make git changes across many repos"
+  homepage "https://github.com/Clever/microplane"
+  url "https://github.com/Clever/microplane/archive/v0.0.21.tar.gz"
+  sha256 "8deb72bdc72f96420decee7b680fa8f5feb48adc39f31e091e44a57734a4a575"
+  license "Apache-2.0"
+  head "https://github.com/Clever/microplane.git"
+
+  depends_on "dep" => :build
+  depends_on "go" => :build
+
+  def install
+    ENV["GOPATH"] = buildpath
+    ENV["GOOS"] = "darwin"
```

Comment:
```suggestion
```
I'm pretty sure you want to compile for the same OS as the host, so setting `GOOS` should be unnecessary.